src/index.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="mainContainer">
+      <header class="head"></header>
+      <section class="container">
+        <div class="todo-wrap" data-form>
+          <h1>Today's To Do...</h1>
+          <div class="btnInput">
+            <input
+              type="text"
+              class="todo-name"
+              id="input-task"
+              data-input
+              placeholder="Add to your list...."
+            />
+            <button class="add-todo">⬅</button>
```

Comment:
- [ ] @elhamatokhi great work so far kindly ensure this button icon matches the one given in the design, also the completed button background doesn't match the design given ,below is a comparison:

Design  Given         |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/213727368-14ba2ef3-b6b9-4948-b5c0-67cce10913bb.png)  | ![image](https://user-images.githubusercontent.com/61408860/213727453-a41a8cfe-16e5-417e-9dc8-c859cf179d7f.png)